.github/workflows/pr.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         with:
           submodules: recursive
       - name: esp-idf build
-        uses: BrianPugh/esp-idf-ci-action@main
+        uses: espressif/esp-idf-ci-action@main
         with:
-          esp_idf_version: v4.2.3
-          action: "make defconfig && make -j8 tests-build"
+          esp_idf_version: v4.3.2
```

Comment:
Would it make sense to add `matrix` to build for multiple versions, above? Something along these lines:
```yml
    strategy:
      matrix:
        idf_ver: ["v4.2.3", "v4.3.3", "v4.4.1", "latest"]
...
        with:
          esp_idf_version: ${{ matrix.idf_ver }}
```

Reply:
addressed in https://github.com/joltwallet/esp_littlefs/pull/91/commits/b286cb8467074f3c5ae6852c45a6b2ba06019376